src/components/CreditCard/CreditCard.jsx

```diff
@@ -0,0 +1,26 @@
+import { type } from "@testing-library/user-event/dist/type"
+
+function CreditCard(props) {
+    const {img, type, number, expirationMonth, expirationYear, bank, owner, bgColor, color} = props;
+    
+  return (
+    <div style={{backgroundColor: `${bgColor}`, color: `${color}`}} class="creditCard">
+        <span class="creditCard-logo">
+            <h6>{type}</h6>
+            <img src={img} alt="credit card" />
+        </span>
+        <span className="creditCard-number">
+            <h4>{number}</h4>
```

Comment:
Here you are just missing rendering multiple "*" followed by the last 4 numbers of the card